docs/src/notebooks/7_community_model.jl

```diff
@@ -0,0 +1,125 @@
+# # Building and analysing a small community model
+
+#md # [![](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/notebooks/@__NAME__.ipynb)
+#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/notebooks/@__NAME__.ipynb)
+
+# Here we will use `COBREXA` to build and analyze a small community model  
+# consisting of three *E. coli* mutants using the `CoreModel`. We will use an
+# objective function that enforces equal growth rates.
+
+# ## Load the base model
+
+!isfile("e_coli_core.json") &&
+    download("http://bigg.ucsd.edu/static/models/e_coli_core.json", "e_coli_core.json");
+
+using COBREXA
+using Tulip
+
+# ## Load the models and inspect fba solutions
+
+base_model = load_model(CoreModel, "e_coli_core.json")
+
+cytbd_knockout = remove_reactions(base_model, "CYTBD")
+sol = flux_balance_analysis_dict(cytbd_knockout, Tulip.Optimizer)
+println("Cytochrome oxidase knockout μ = ", sol["BIOMASS_Ecoli_core_w_GAM"])
+
+atps4r_knockout = remove_reactions(base_model, "ATPS4r")
+sol = flux_balance_analysis_dict(atps4r_knockout, Tulip.Optimizer)
+println("ATP synthase knockout μ = ", sol["BIOMASS_Ecoli_core_w_GAM"])
+
+eno_knockout = remove_reactions(base_model, "ENO")
+sol = flux_balance_analysis_dict(eno_knockout, Tulip.Optimizer)
+println("Enolase knockout μ = ", sol["BIOMASS_Ecoli_core_w_GAM"])
+# notice that the enolase mutant cannot grow by itself.
+
+# ## Build a community model of the Cytochrome oxidase and ATP synthase knockouts 
+
+ex_rxns = filter(looks_like_exchange_reaction, reactions(base_model))
+ex_mets = [first(keys(reaction_stoichiometry(base_model, ex_rxn))) for ex_rxn in ex_rxns]
```

Comment:
needs a comment -- "Extract the metabolite indexes that correspond to the exchange reactions"